packages/story-editor/src/components/videoTrim/videoTrimmer.js

```diff
@@ -55,38 +48,95 @@ function VideoTrimmer() {
     maxOffset,
     setStartOffset,
     setEndOffset,
+    trimBackgroundImage,
```

Comment:
Nit: this sounds like an action by name.

Reply:
Will remove - was some earlier work to be fixed in #9104.

Reply:
Fixed in 1dbb0f9951.